winui/Kanban/column.md

```diff
@@ -0,0 +1,172 @@
+
+{% tabs %}
+{% highlight XAML hl_lines="3 4" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 MinimumColumnWidth="300"
+                 MinimumColumnWidth="350">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.MinimumColumnWidth = 300;
+this.kanban.MaximumColumnWidth = 350;
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also define the exact width for the columns using `ColumnWidth` property of `SfKanban`.
+
+{% tabs %}
+{% highlight XAML hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 ColumnWidth="400">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+this.kanban.ColumnWidth = 400;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Categorizing columns
+
+In the WinUI Kanban Control, cards are categorized and placed into their respective columns based on the value of the `Category` property of `KanbanModel`.
+
+### Populating columns with cards from Multiple categories
+
+You can populate a single column with cards from multiple categories by assigning multiple values to the `Categories` collection of the `KanbanColumn`. For example, to group tasks categorized as `Done` and `Validated` under a single `Done`column, you can include both category values in the `Categories` collection.
+
+{% tabs %}
+{% highlight XAML hl_lines="2 7" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 AutoGenerateColumns="False" 
+                 ItemsSource="{Binding TaskDetails}">
+        <kanban:KanbanColumn HeaderText="To Do" Categories="Open" />
+        <kanban:KanbanColumn HeaderText="In Progress" Categories="In Progress" />
+        <kanban:KanbanColumn HeaderText="Code Review" Categories="Code Review" />
+        <kanban:KanbanColumn HeaderText="Done" Categories="Done, Validated" />
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "To Do", Categories = "Open" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "In Progress", Categories = "In Progress" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Code Review", Categories = "Code Review" });
+this.kanban.Columns.Add(new KanbanColumn() { HeaderText = "Done", Categories = "Done, Validated" });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing column headers
+
+### Customizing column header text
+
+You can set the header text for a column using the `HeaderText` property, which allows you to display simple text as the column title.
```

Comment:
You can use the `HeaderText` property of `KanbanColumn` to customize the header text of a column in the kanban control.

Reply:
Changed content